teos/src/watcher.rs

```diff
@@ -705,11 +705,30 @@ impl Watcher {
         self.dbm.lock().unwrap().load_all_appointments()
     }
 
+    /// Gets all the appointments matching a specific locator from the [Watcher] (from the database).
+    pub(crate) fn get_watcher_appointments_with_locator(
+        &self,
+        locator: Locator,
+    ) -> HashMap<UUID, ExtendedAppointment> {
+        self.dbm
+            .lock()
+            .unwrap()
+            .load_appointments_with_locator(locator)
```

Comment:
I looked at some parts of the codebase using `load_all_appointments` (like this [one](https://github.com/talaia-labs/rust-teos/blob/master/teos/src/watcher.rs#L272)) and imagined it would make more sense to have the method names descriptive enough as they are. The indirection can still be removed anyway since `load_appointments` has a docstring describing its functionality. I prefer the old names though.

Reply:
Yeah, I do agree. Seems like there are too many load appointment / tracker methods now.

Reply:
Did you decide to stick with the additional indirection in the end?

Reply:
> That said, it's ok to remove the indirection since load_appointments has a descriptive docstring anyways.
I thought your comment was in favor of leaving the method names as they are, but since you favor the load_appointments way, I will remove the indirection in the next patch.

Oh no, no worries really, it's up to you at the end of the day. I think both solutions are acceptable.